### DISCUSSION TITLE
[Feature] `CalendarDateRangePicker` control

### DISCUSSION THREAD

**Discussion by xperiandri** [CommunityToolkit/Labs-Windows#474] [Ideas]
### Describe the problem

Control to select a range of dates

### Describe the solution

`CalendarDatePicker` control that behaves as RangeSelector but for dates

### Alternatives

Use 2 `CalendarDatePicker` controls

### Help us help you

Maybe later

**Comment by ghost** [comment]
Hello, 'xperiandri! Thanks for submitting a new feature request. I've automatically added a vote 👍 reaction to help get things started. Other community members can vote to help us prioritize this feature in the future!
